ftests/parallel/test_remote.py

```diff
@@ -129,15 +129,21 @@ def test_dsoln(self):
         from nose.plugins.skip import SkipTest
         if sys.platform.startswith('win'): raise SkipTest
         from numpy import zeros
-        from solvcon.batch import Localhost
         from solvcon.domain import Distributed
+        if "SOLVCON_NODELIST" in os.environ:
```

Comment:
test_dsoln_cluster should use a standalone test case.  A test case should behave consistently even when the environment changes.